Libraries/Components/WebView/WebView.android.js

```diff
@@ -58,6 +58,8 @@ class WebView extends React.Component {
     automaticallyAdjustContentInsets: PropTypes.bool,
     contentInset: EdgeInsetsPropType,
     onNavigationStateChange: PropTypes.func,
+    messagingEnabled: PropTypes.bool,
```

Comment:
Adding it here means it's part of the public API. Even if you manage not to show it in docs it's gonna show up in autocompletion.

Why not detect if there's a `onMessage` prop and enable it automatically on native side (or maybe inside render).